Marlin/Configuration_adv.h

```diff
@@ -604,6 +604,11 @@
   #define Z_STEPPER_ALIGN_ITERATIONS 3
   // Enable to restore leveling setup after operation
   #define RESTORE_LEVELING_AFTER_G34
+  
+  // 0.05 is a 5% incline. On a 300mm bed that would be a misalignment of about 1.5cm.
+  // This angle is the maximum misalignment catered for
+  #define G34_MAX_INCLINE_PERCENT 0.05f
```

Comment:
If it is named "percent" then the units should probably be percentage units - in this case, `5.0`. Then divide it by 100 on usage.